static/app/views/performance/newTraceDetails/index.tsx

```diff
@@ -245,7 +230,7 @@ const FlexBox = styled('div')`
 `;
 
 const TraceInnerLayout = styled(FlexBox)`
-  flex: 1 1 100%;
   padding: ${space(2)} ${space(3)};
+  flex-grow: 1;
```

Comment:
@Abdkhan14 without this, the waterfall with few spans doesn't full expand, not sure if that's desired or not, but this more closely matches the previous behaviour